site.yml

```diff
@@ -79,3 +79,26 @@
   - { role: docker, etcd_client_port1: 2379 }
   - { role: swarm, run_as: master }
   - { role: contiv_network, run_as: master }
+
+# Netplugin master
```

Comment:
I'm changing demo installer in contiv/demo/net directory to use netplugin-master and netplugin-worker host-groups.
If you make all hosts as netplugin-node, it starts swarm manager on all hosts and swarm manager fails to come up.